airbyte-webapp/src/components/connection/CatalogTree/StreamHeader.module.scss

```diff
@@ -1,6 +1,6 @@
 @use "scss/colors";
 @use "scss/variables";
-@forward "./CatalogTreeBody.module.scss";
+@forward "./CatalogTreeHeader.module.scss";
```

Comment:
This was causing the issue with the plus and minus icons being shifted. Caused by the changes in #18089